src/HttpClient/Guzzle.php

```diff
@@ -43,6 +44,13 @@ class Guzzle implements HttpClientInterface
      */
     public $userAgent = 'Opauth';
 
+    /**
+     * Set `Content-Type: application/x-www-form-urlencoded` for POST request
+     *
+     * @var bool
+     */
+    private $formUrlencoded = false;
```

Comment:
Let's change this to `$contentType`. It would leave space for more options in the future, and would avoid us having to introduce even more properties in the future.

```suggestion
    private $contentType = \GuzzleHttp\RequestOptions::JSON;
```

The method can then be renamed to `setContentType()` and accept one of the two `RequestOptions::*` constants. It could also throw an `InvalidArgumentException` for any value that isn't either of them.

Reply:
Ah it makes more sense and it'll be more flexible :) Updated now

Reply:
Tidy 👌